src/Workspaces/Core/Portable/FindSymbols/Shared/AbstractSyntaxIndex_Persistence.cs

```diff
@@ -59,7 +83,7 @@ internal partial class AbstractSyntaxIndex<TIndex> : IObjectWritable
             return null;
         }
 
-        public static async Task<Checksum> GetChecksumAsync(
+        public static async ValueTask<(Checksum textOnlyChecksum, Checksum textAndDirectivesChecksum)> GetChecksumsAsync(
```

Comment:
might be unrelated to this PR, but do we have unit tests for this system at all?  For example verifying that we get all the expected document checksums if we give it a project, making changes and verifying checksums are updated, etc?